known.json

```diff
@@ -13,11 +13,12 @@
 	"14175575863689886451L"	: { "owner": "Adviser Assets", "description": "LiskHQ" },
 	"5726759782318848681L"	: { "owner": "Lisk Foundation", "description": "LiskHQ" },
 	"15841793714384967784L"	: { "owner": "Lisk Community Assets", "description": "LiskHQ" },
-	"3040783849904107057L"	: { "owner": "Coincheck.com", "description": "Old Wallet" },
+	"3040783849904107057L"	: { "owner": "Coincheck.com", "description": "Old old Wallet" },
```

Comment:
I'm open to wording this differently 😅

Reply:
I'm not an expert of English language, but wouldn't just `Former (hot/cold/main) wallet` sound better? :)
Also, it would make sense to merge that change with the branch `1.6.1`. It seems to be released sooner than `2.0.0` and it'll be part of 2.1.0 in the future.